vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -1932,13 +1932,13 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
                             VsShellUtilities.ShowMessageBox
                                 (
                                     node.Site, 
-                                    FSharpSR.GetString(FSharpSR.Dev11SupportsOnlySilverlight5), 
+                                    FSharpSR.Dev11SupportsOnlySilverlight5(),
                                     null,
                                     OLEMSGICON.OLEMSGICON_INFO, OLEMSGBUTTON.OLEMSGBUTTON_OK, OLEMSGDEFBUTTON.OLEMSGDEFBUTTON_FIRST
                                 ) |> ignore
                             Marshal.ThrowExceptionForHR(VSConstants.OLE_E_PROMPTSAVECANCELLED)
                         let result =
-                            VsShellUtilities.ShowMessageBox(node.Site, FSharpSR.GetStringWithCR(FSharpSR.NeedReloadToChangeTargetFx), 
+                            VsShellUtilities.ShowMessageBox(node.Site, FSharpSR.NeedReloadToChangeTargetFx().Replace("\n", Environment.NewLine),
```

Comment:
This is incorrect - this should be `@"\n"`.

Reply:
I don't know how many times I have made that same mistake over the years.  Good eyes.